lib/createWorker.js

```diff
@@ -28,32 +27,18 @@ function formatCreateWorkersParameters(handler, config, options) {
  * Create a worker instance based on configuration
  * @param {function} handler function to handle each message
  * @param {Object} config configuration for the worker
- * @param {String} config.workerName name for the worker
- * @param {String} config.amqpUrl url to amqp server
- * @param {String} config.exchangeName name of the exchange to use
- * @param {String} config.queueName name of the queue to listen on
- * @param {String} [config.routingKey] name of the routing key to bind to
  * @param {Object} [options] additional options
- * @param {function} [options.validator] function to validate the message with
- * @param {Number} [options.heartbeat] to override default heartbeat
- * @param {Number} [options.taskTimeout] to override default task timeout
- * @param {Number} [options.processExitTimeout] to override default process exit timeout
- * @param {Number} [options.channelPrefetch] to override default channel prefetch value
  * @returns {Object} a worker instance with connection, channel, and listen/close functions
- * @deprecated
+ * @deprecated use createWorkers instead
  */
 function createWorker(handler, config, options = {}) {
```

Comment:
What do you think if we completely remove this legacy function and bump a major version?